Tests/MongoSwiftTests/MongoClientTests.swift

```diff
@@ -82,6 +82,22 @@ final class MongoClientTests: MongoSwiftTestCase {
         expect(ids[2]).to(equal(ids[1] + 1))
     }
 
+    func testBound() throws {
+        let elg = MultiThreadedEventLoopGroup(numberOfThreads: 4)
+        let expectedEventLoop = elg.next()
+        try self.withTestClient { client in
```

Comment:
I don't see a way to directly check whether an event loop is a member of a group. however, I'm not sure that is a problem, can you think of any issues you might run into allowing an arbitrary event loop?

Reply:
I wasn't sure if there were any NIO invariants that would be violated by hopping between loops in different groups, though I don't really have anything in mind / can't think of anything.

Reply:
I'm not aware of any potential issues with that. I think the ELG is just a convenience for multiplexing across event loops and there's no relationship between the individual event loops otherwise.